queue_services/entity-emailer/tests/unit/email_processors/test_continuation_out_notification.py

```diff
@@ -0,0 +1,63 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Unit Tests for the Continuation Out email processor."""
+from unittest.mock import patch
+
+import pytest
+from legal_api.models import Business
+
+from entity_emailer.email_processors import continuation_out_notification
+from tests.unit import prep_continuation_out_filing
+
+
+@pytest.mark.parametrize('status,legal_type,submitter_role', [
+    ('COMPLETED', Business.LegalTypes.COMP.value, None),
+    ('COMPLETED', Business.LegalTypes.BCOMP.value, None),
+    ('COMPLETED', Business.LegalTypes.BC_CCC.value, None),
+    ('COMPLETED', Business.LegalTypes.BC_ULC_COMPANY.value, None),
+    ('COMPLETED', Business.LegalTypes.COMP.value, 'staff'),
+    ('COMPLETED', Business.LegalTypes.BCOMP.value, 'staff'),
+    ('COMPLETED', Business.LegalTypes.BC_CCC.value, 'staff'),
+    ('COMPLETED', Business.LegalTypes.BC_ULC_COMPANY.value, 'staff')
+])
+def test_continuation_out_notification(status, legal_type, submitter_role):
```

Comment:
I think to get your tests to pass, you just need to add app, session, to the start of your parameters for this function